storage/signed_urls/generate_signed_urls.py

```diff
@@ -88,6 +90,7 @@ def generate_signed_url(service_account_file, bucket_name, object_name,
         lower_k = str(k).lower()
```

Comment:
Similar issue with names. Also, these aren't signed_headers, they're names of the headers that need to be signed, so a different name would be more clear. And no need to use items() since you just want the keys, no values.